tests/e2e/step_delegation.go

```diff
@@ -368,9 +263,9 @@ func stepsRedelegateShort(consumerName string) []Step {
 				ChainID(consumerName): ChainState{
 					ValPowers: &map[ValidatorID]uint{
 						// Now power changes are seen by consumer
-						ValidatorID("alice"): 509,
+						ValidatorID("alice"): 910,
```

Comment:
Why has the ratio changed so much?

Are you trying to check that the soft opt-out no longer works?

Reply:
> Why has the ratio changed so much?

TL;DR because we removed opt-out related steps that were reducing the `ValPower` of `alice`.

Specifically, we removed the `stepsRedelegateForOptOut` and `stepsDowntimeWithOptOut`. The `stepsRedelegateForOptOut` was the one that [reduced](https://github.com/cosmos/interchain-security/blob/060c943b2f17b2696e668c669352a366d3610d5d/tests/e2e/step_delegation.go#L223) the `ValPower` of `alice` to `60` so it could test the opt out functionality.

Before, in the `happyPathSteps` we had:
```
...
stepsRedelegateForOptOut("consu"),
stepsDowntimeWithOptOut("consu"),
stepsRedelegate("consu"),
...
```
but we removed `stepsRedelegateForOptOut` and `stepsDowntimeWithOptOut` and hence now when `stepsRedelegate` is called, the `ValPower` of `alice` is not `60` anymore but `510`. Redelegating `400` to `alice` makes `alice` have a `ValPower` `910`.

> Are you trying to check that the soft opt-out no longer works?
No.

Reply:
Thank you for elaborating. It was not clear to me after going over it a couple times.